lib/presentation/crowdaction/crowdaction_details/crowdaction_details_screen.dart

```diff
@@ -279,6 +245,18 @@ class _CrowdActionDetailsPageState extends State<CrowdActionDetailsPage> {
     );
   }
 
+  Shimmer _imagePlaceholder(BuildContext context) {
```

Comment:
Extract widget and use for `crowdaction_card.dart` and `micro_crowdaction_card.dart` as well.

Reply:
It's a fixed height so it won't apply to other screens.

Reply:
We can pass the height as a parameter to the custom widget.

Reply:
Cool ... let me do that